doctr/datasets/utils.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import string
+import unicodedata
+import numpy as np
+from typing import Dict, List
+
+__all__ = ['translate', 'encode_sequence', 'decode_sequence', 'VOCABS']
+
+
+VOCABS: Dict[str, str] = {
+    'digits': string.digits,
+    'ascii_letters': string.ascii_letters,
+    'punctuation': string.punctuation,
+    'currency': '£€¥¢฿',
+    'latin': string.digits + string.ascii_letters + string.punctuation + '°',
+    'french': string.digits + string.ascii_letters + string.punctuation + '°' + 'àâéèêëîïôùûçÀÂÉÈËÎÏÔÙÛÇ' + '£€¥¢฿',
+}
```

Comment:
Perhaps we should move this to `datasets/vocabs.py`?

This will most likely be extended often in the future (adding language specific characters, math symbols, etc.), perhaps more often then utils.py